testbed/tests/trace_test.go

```diff
@@ -349,8 +364,15 @@ func TestTraceAttributesProcessor(t *testing.T) {
 			// Create a service name that matches "include" filter.
 			nodeToInclude := &commonpb.Node{ServiceInfo: &commonpb.ServiceInfo{Name: "service-to-add-attr"}}
 
-			verifySingleSpan(t, tc, nodeToInclude, spanToInclude, func(span *tracepb.Span) {
-				// Verify attributes was added.
+			verifySingleSpan(t, tc, nodeToInclude, spanToInclude, func(span data.Span) {
```

Comment:
With 2 lambda funcs this call is a bit difficult to understand. Perhaps turn lambdas into proper named functions?
Also it would be worth explaining why we have 2 verification functions.

Reply:
done